Sources/Sentry/SentryPropagationContext.h

```diff
@@ -1,19 +1,29 @@
 #import <Foundation/Foundation.h>
 
-@class SentryId, SentrySpanId, SentryTraceContext, SentryTraceHeader;
+@class SentryId, SentrySpanId, SentryTraceContext, SentryTraceHeader, SentryOptions, SentryScope;
 
 NS_ASSUME_NONNULL_BEGIN
 
 @interface SentryPropagationContext : NSObject
 
 @property (nonatomic, strong) SentryId *traceId;
 @property (nonatomic, strong) SentrySpanId *spanId;
-@property (nonatomic, readonly, nullable) SentryTraceContext *traceContext;
```

Comment:
Let's please stop using properties to autosynthesize methods that never store a backing ivar, it's confusing and I don't believe what they were intended for.